apis/infix-en_GB/atrium-api-infix-en_GB-jvm/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/pathAssertions.kt

```diff
@@ -298,3 +300,55 @@ val <T : Path> Expect<T>.extension: Expect<String>
  */
 infix fun <T : Path> Expect<T>.extension(assertionCreator: Expect<String>.() -> Unit): Expect<T> =
     _logic.extension().addToInitial(assertionCreator)
+
+/**
+ * Expects that the subject of the assertion (a [Path]) has the same textual content
+ * as [targetPath].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+infix fun <T : Path> Expect<T>.hasSameTextualContentAs(
+    targetPath: Path
+): Expect<T> = hasSameTextualContentAs(withEncoding(targetPath, this))
+
+/**
+ * Expects that the subject of the assertion (a [Path]) has the same textual content
+ * as path in [encodingWithCreator].
+ *
+ *  Use the function `withEncoding(Path, Charset, Charset) { ... }` to create an [EncodingWithCreator].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+infix fun <T : Path> Expect<T>.hasSameTextualContentAs(encodingWithCreator: EncodingWithCreator<T>):
+    Expect<T> = _logicAppend {
+        hasSameTextualContentAs(
+            encodingWithCreator.path, encodingWithCreator.sourceCharset, encodingWithCreator.targetCharset
+        )
+    }
+/**
+ * Expects that the subject of the assertion (a [Path]) has the same binary content
+ * as [targetPath].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+infix fun <T : Path> Expect<T>.hasSameBinaryContentAs(targetPath: Path):
+    Expect<T> = _logicAppend {
+    hasSameBinaryContentAs(targetPath)
+}
```

Comment:
please don't put the return type on a new line
```suggestion
infix fun <T : Path> Expect<T>.hasSameBinaryContentAs(targetPath: Path): Expect<T> = 
    _logicAppend { hasSameBinaryContentAs(targetPath) }
```